cmd/config/env_var_reader.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/flyteorg/flyte/flyteidl/clients/go/admin"
+	"github.com/flyteorg/flyte/flytestdlib/config"
+)
+
+const flyteAdminEndpoint = "FLYTE_ADMIN_ENDPOINT"
```

Comment:
@pingsutw do we typically follow the pattern of ending with "EnvVar"? Ex: flyteAdminEndpointEnvVar